ic-agent/src/agent/http_transport/reqwest_transport.rs

```diff
@@ -7,29 +7,28 @@ pub use reqwest;
 use futures_util::StreamExt;
 use reqwest::{
     header::{HeaderMap, CONTENT_TYPE},
-    Body, Client, Method, Request, StatusCode, Url,
+    Body, Client, Method, Request, StatusCode,
 };
 
 use crate::{
     agent::{
         agent_error::HttpErrorPayload,
-        http_transport::{IC0_DOMAIN, IC0_SUB_DOMAIN},
+        http_transport::route_provider::{RoundRobinRouteProvider, RouteProvider},
         AgentFuture, Transport,
     },
     export::Principal,
     AgentError, RequestId,
 };
 
 /// A [`Transport`] using [`reqwest`] to make HTTP calls to the Internet Computer.
-#[derive(Debug)]
```

Comment:
was this intentional?

Reply:
accidental, thanks for spotting, changed back